packages/contracts/contracts/plugin/psp/utils/Common.sol

```diff
@@ -62,7 +68,9 @@ function hashHelpers(address[] memory _helpers) pure returns (bytes32) {
 
 /// @notice Returns a hash of an array of multi-targeted permission operations.
 /// @param _permissions The array of of multi-targeted permission operations.
-/// @return bytes The hash of the array of permission operations.
-function hashPermissions(PermissionLib.MultiTargetPermission[] memory _permissions) pure returns (bytes32) {
+/// @return The hash of the array of permission operations.
+function hashPermissions(
```

Comment:
`hashPermission`  looks very similar to `hasPermission`. 

Would `hashedPermission` look more different? Or `getPermissionHash()`

Reply:
`getPermissionHash` is bad because we have exactly this function on the PermissionManager

Reply:
`hashPermissions`
vs.
`hasPermission`
is tolerable.

Otherwise, I would propose `hashPermissionArray`

Reply:
Let's leave it as is :+1: